common-npm-packages/artifacts-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tasks-artifacts-common",
-  "version": "1.178.0",
+  "version": "1.179.0",
```

Comment:
I think we do need to bump up all the tasks that depend on artifacts-common as well for this change to actually take affect on the tasks (this is what you asked before, and I gave an answer to a different question, whoops)

Reply:
yes and no.  The packages inside common-npm-packages get pushed to the npm registry now, which then means that different tasks can uprev dependencies at their own pace.  The current CI/CD to push to npm is broken so its done manually by working with Danny.  After its pushed, the dependent task(s) can then get updated to pick up the new version of the common package.